blocks/init/src/Blocks/components/accordion/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-	"$schema": "./../../../../node_modules/@eightshift/frontend-libs/schemas/component.json",
+	"$schema": "https://raw.githubusercontent.com/infinum/eightshift-frontend-libs/develop/schemas/component.json",
```

Comment:
Just mark this as a TODO - the URL should point to the main branch, as that will be the stable branch, develop branch can have changes on it, so it wouldn't be good to check the schema against that branch.